ibm/mas_devops/roles/dro/tasks/uninstall/main.yml

```diff
@@ -0,0 +1,129 @@
+---
+# 1. Delete CR, CSV and Subscription
+# -----------------------------------------------------------------------------
+# Delete the MarketplaceConfig
+- name: "uninstall : Delete the MarketplaceConfig"
```

Comment:
Do we want to delete the marketplace config? what if the user had redhat marketplace already installed?

Reply:
MarketplaceConfig is created and managed by ibm-metrics-operator. There are no specific labels/annotations to identify if the MarketplaceConfig is installed by user before MAS installation or is created by MAS. 
I'll try to add some kind of check to identify existing MarketplaceConfig.